tests/components/epic_games_store/const.py

```diff
@@ -0,0 +1,1613 @@
+"""Test constants."""
```

Comment:
I think some of these constants should be moved to json fixtures.

Reply:
I don't know how to do json fixtures, do you have an exemple ?
Thanks

Reply:
You just need to create a sub folder `tests/components/epic_games_store/fixtures`, for example with `data_error_wrong_country.json`

Then you can use `from tests.common import load_json_object_fixture`